integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -1130,6 +1133,133 @@ public void testRestartingMSWithExplicitServerStartStateWhileVaryingReplicaCount
     logger.info("managed server " + serverName + " restarted successfully.");
   }
 
+  /**
+   * Scale the configured cluster using the sample script scaleCluster.sh script
+   * Verify that server(s) in the configured cluster are scaled up and in RUNNING state.
+   * Verify that server(s) in the dynamic cluster are not affected.
+   * Restore the env using the sample script stopServer.sh.
+   */
+  @Order(19)
+  @Test
+  @DisplayName("Scale the configured cluster with scaleCluster.sh script")
+  public void testConfigClusterScale() {
+    int newReplicaCount = 2;
+    String configServerName = "config-cluster-server" + newReplicaCount;
+    String configServerPodName = domainUid + "-" + configServerName;
+    String dynamicServerPodName = domainUid + "-managed-server" + newReplicaCount;
+
+    // make sure the server-to-be-test doesn't exist
+    restoreEnv();
+    checkPodDoesNotExist(configServerPodName, domainUid, domainNamespace);
+    checkPodDoesNotExist(dynamicServerPodName, domainUid, domainNamespace);
+
+    // use scaleCluster.sh to scale a configured cluster
+    logger.info("Scale configured cluster using the script scaleCluster.sh");
+    String result =  assertDoesNotThrow(() ->
+        executeLifecycleScript(SCALE_CLUSTER_SCRIPT, CLUSTER_LIFECYCLE, CLUSTER_2, " -r " + newReplicaCount, false),
+        String.format("Failed to run %s", SCALE_CLUSTER_SCRIPT));
+
+    checkPodReadyAndServiceExists(configServerPodName, domainUid, domainNamespace);
+    assertDoesNotThrow(() -> assertTrue(checkClusterReplicaCountMatches(CLUSTER_2,
+        domainUid, domainNamespace, newReplicaCount)));
+
+    // use clusterStatus.sh to verify scaling results
+    result =  assertDoesNotThrow(() ->
+        executeLifecycleScript(STATUS_CLUSTER_SCRIPT, CLUSTER_LIFECYCLE, CLUSTER_2),
+        String.format("Failed to run %s", STATUS_CLUSTER_SCRIPT));
+
+    String regex = ".*" + CLUSTER_2 + "(\\s+)0(\\s+)2(\\s+)2(\\s+)2(\\s+)2";
+    assertTrue(verifyExecuteResult(result, regex),"The script should scale up the cluster: " + CLUSTER_2);
+    logger.info("Configured cluster " + CLUSTER_2 + " scaled successfully.");
+
+    // check managed server from other cluster are not affected
+    logger.info("Check dynamic managed server pods are not affected");
+    assertDoesNotThrow(() -> assertTrue(checkClusterReplicaCountMatches(CLUSTER_1,
+        domainUid, domainNamespace, replicaCount)));
+
+    checkPodDoesNotExist(dynamicServerPodName, domainUid, domainNamespace);
+
+    // restore test env
+    result =  assertDoesNotThrow(() ->
+        executeLifecycleScript(STOP_SERVER_SCRIPT, SERVER_LIFECYCLE, configServerName),
```

Comment:
Can you use the same script scaleCluster.sh to scale down the cluster instead of stopServer.sh script

Reply:
good idea. Wii change

Reply:
changed